### DISCUSSION TITLE
Single column update if the table is bidirectional

### DISCUSSION THREAD

**Discussion by maxrodro** [Mimetis/Dotmim.Sync#978] [Q&A]
hi, @Mimetis 

Is there a way to update only one column if the table is bidireccional?

Example

Table: Employees

ID,  
code, -> Ignore changes
Name, -> Ignore changes
IsActive, -> Ignore changes
...,   -> Ignore changes
IsConnected --> I only want to allow updating this column on the server, ignore the other changes to the other columns

It's possible?

**Comment by SvdSinner** [comment]
There are instructions on how to filter columns at https://dotmimsync.readthedocs.io/Configuration.html

**Reply by maxrodro** [reply]
Hi @Mimetis 

Any comments from you on this topic?

**Reply by Mimetis** [reply]
Really depends on your scenario honestly

If you have only one table to sync, where you want only 4 columns without `IsConnected`, just create the right `SyncSetup `where you are adding table and columns name
That way, your client will never receive the `IsConnected` column, and of course will never send back any update on this column.

Another scenario, more complex is when you want this column to be synced from server to client, but not client to server
In this typical scenario, depending on your need, you can.
- Use to scopes: One `DownloadOnly` with all columns, and one `UploadOnly `without the column
- Use only one default scope (`BiDirectional`) and tweak the update stored procedure to be sure the value will never override the server value. Easier, but you need to tweak the stored procedure creation.

I don't see any other scenario, let me know if you think about something else.

**Comment by maxrodro** [comment]
Thank you very much for your options, I will discuss it internally
